iet.so/src/action.cpp

```diff
@@ -127,12 +128,13 @@ using std::fstream;
 #define IET_DEFAULT_CP_WORKLOAD         true
 #define IET_DEFAULT_HOT_CALLS           1
 #define IET_DEFAULT_MATRIX_INIT         "default"
-#define IET_DEFAULT_GEMM_MODE        ""
+#define IET_DEFAULT_GEMM_MODE           ""
 #define IET_DEFAULT_BATCH_SIZE          0
 #define IET_DEFAULT_STRIDE_A            0
 #define IET_DEFAULT_STRIDE_B            0
 #define IET_DEFAULT_STRIDE_C            0
 #define IET_DEFAULT_STRIDE_D            0
+#define IET_DEFAULT_WG_COUNT            80
```

Comment:
80 is for a specific ASIC ? if that is the case we must put correct value here if no config param is present. hipDeviceProp_t  has maxThreadsPerBlock which is the maximum WG size. Please use this